docs/checkers/index.md

```diff
@@ -2965,6 +2965,17 @@ for all lines is 80 characters.
 
 ```
 
+(W3601)=
+
+### Bad chained comparison (W3601)
+
+This error occurs when a chained comparison uses semantically incompatible operators. For example,
```

Comment:
Please rephrase this for beginners. Don't use the word "semantically"---this isn't the pylint documentation. Also provide more examples for different cases.